### ISSUE TITLE
Debian testing: Brave loads slow on lichess.org

### ISSUE THREAD

**Issue by M4rt1n12** [brave/brave-browser#39150]
### Description

Sometimes lichess.org website loads slow (30 sec) or hangs. It is NOT an Internet Connection issue! Firefox works fine.
Debian testing with kernel 6.6 and latest brave version (deb package)

### Steps to reproduce

1. 
2. 
3. 

### Actual result

Load lichess.org in about 30 seconds, interface hangs

### Expected result

Load website within seconds

### Reproduces how often

Intermittent issue

### Brave version (brave://version info)

Version 1.67.116 Chromium: 126.0.6478.71 (Offizieller Build) (64-Bit)

### Channel information

- [X] release (stable)
- [ ] beta
- [ ] nightly

### Reproducibility

- [ ] with Brave Shields disabled
- [ ] with Brave Rewards disabled
- [ ] in the latest version of Chrome

### Miscellaneous information

Occoured since a few weeks.

**Comment by rebron** [comment]
@M4rt1n12 Any extensions running or custom shields settings for lichess.org. Looks like its loading pretty quickly for me.

**Comment by M4rt1n12** [comment]
No, I don't have any extensions. Right now lichess is loading fast on my machine too and it hasn't happened in the last 2 days, but I noticed it in the weeks before that.

**Comment by M4rt1n12** [comment]
Now it's slowly opening: https://seaworld.com/orlando/animal-experiences
It might have something to do with the uptime of the OS.

**Comment by M4rt1n12** [comment]
Another website that is hanging very often: https://www.pineca.de

**Comment by M4rt1n12** [comment]
it is an OS issue

**Comment by M4rt1n12** [comment]
I found the solution. It was a static smb connection in my /etc/fstab that connects a remote NAS via wireguard.

**Comment by M4rt1n12** [comment]
It occoured again, somehow again only with brave. I regulary get „ERR_CONNECTION_RESET“ when opening lichess.org. Also I can only open the context menu in Brave (on all websites) when I do a right double click .
